svm30/Makefile

```diff
@@ -0,0 +1,52 @@
+sht_source_dir := ../../embedded-sht/shtc1
+sht_common_dir := ../../embedded-sht/sht-common
+sht_sources := ${sht_source_dir}/shtc1.c ${sht_common_dir}/sht.h \
+               ${sht_common_dir}/sht_common.c ${sht_common_dir}/sht_common.h \
+               ${sht_common_dir}/git_version.c ${sht_common_dir}/git_version.h
+
+sgp_source_dir := ../sgp30
+sgp_common_dir := ../sgp-common
+sgp_sources := ${sgp_source_dir}/sgp30_featureset.c \
+               ${sgp_common_dir}/sgp_featureset.h \
+               ${sgp_common_dir}/sgp_git_version.c ${sgp_common_dir}/sgp_git_version.h \
+               ${sgp_source_dir}/sgp30.c ${sgp_source_dir}/sgp30.h
+
+sensirion_common_dir := ../embedded-common
+sensirion_common_sources := ${sensirion_common_dir}/sensirion_arch_config.h \
+                            ${sensirion_common_dir}/sensirion_i2c.h \
+                            ${sensirion_common_dir}/sensirion_common.c
+
+sw_i2c_dir := ${sensirion_common_dir}/sw_i2c
+hw_i2c_dir := ${sensirion_common_dir}/hw_i2c
+
+CFLAGS := -Os -Wall -Werror -I. -I${sensirion_common_dir} \
```

Comment:
I think we should avoid using `-WError` as a default, since this might brake Makefiles due to gcc updates. We should only use `-WError` for testing

Reply:
Thanks, I agree and I'll create a follow-up commit to remove those in all instances. The immediate breaking risk is rather low atm. due to the pinned gcc-5.4 docker image.

Reply:
yes I agree for the docker image, its more if somebody want's to build it on their computer. It should not fail just because they use a different gcc